DataFormats/TotemRawData/interface/TotemRawToDigiStatus.h

```diff
@@ -0,0 +1,85 @@
+/****************************************************************************
```

Comment:
This file (named TotemRawToDigiStatus) contains
 ``` class TotemVFATStatus  ```
and a typedef
``` typedef std::map<TotemStructuralVFATId, TotemVFATStatus> TotemRawToDigiStatus;  ```

It is more common to match the name of the file with the class defined,
rather than the derived typedef.